autosklearn/automl.py

```diff
@@ -309,6 +309,9 @@ def _do_dummy_prediction(self, datamanager, num_run):
         else:
             self._logger.error('Error creating dummy predictions: %s ',
                                str(additional_info))
+            # Fail if dummy prediction fails.
+            # Should we provide more details?
+            raise ValueError("Dummy prediction failed!")
```

Comment:
Could you please add additional information to the value error as done above in the logging and also add a unit test for this?